client/web/src/enterprise/batches/create/CreateOrEditBatchChangePage.tsx

```diff
@@ -352,6 +352,10 @@ const EditPage: React.FunctionComponent<EditPageProps> = ({
             <div className={classNames(styles.editorLayoutContainer, 'd-flex flex-1')}>
                 <LibraryPane name={batchChange.name} onReplaceItem={clearErrorsAndHandleCodeChange} />
                 <div className={styles.editorContainer}>
+                    {/* <animated.h4 className=“overflow-hidden” style={headerStyle}>
+                        Library
+                    </animated.h4>                    */}
+                    <h4 className={styles.header}>Batch specification</h4>
```

Comment:
```suggestion
                    <h4 className={styles.header}>Batch spec</h4>
```

Seems like the designs are still kinda outdated 😬 This is the wording we want to go with